source/docs/transportation/alt-fuel-stations-v1/_request_params_common2.md.erb

```diff
@@ -411,7 +411,7 @@
       <strong>Default:</strong> <em>all</em>
     </div>
     <div class="doc-parameter-value-field">
-      <strong>Options:</strong> <em>all, AeroVironment Network, Blink Network, ChargePoint Network, EV Connect, eVgo Network, EVSE LLC WebNet, Greenlots, OpConnect, RechargeAccess, SemaCharge Network, Shorepower, Tesla</em>
+      <strong>Options:</strong> <em>all, AeroVironment Network, Blink Network, ChargePoint Network, EV Connect, eVgo Network, GE WattStation, Greenlots, OpConnect, SemaCharge Network, Tesla</em>
```

Comment:
@edatrix: Ah, did @reger talk to you about these changes? We had briefly discussed this, but in any case, I don't think we actually want to make these changes to the `ev_network` docs. Since these are lookup codes that we can't change without breaking backwards compatibility, we need to keep the values the same (I think ICF was just looking to tidy the display values, but since these are really more of coded lookup values, they have to remain constant).